candig/server/frontend.py

```diff
@@ -249,35 +247,27 @@ def __init__(self):
         if not self.file_path:
             raise exceptions.ConfigurationException("No user access list defined")
 
+        self.access_list = pd.read_csv(self.file_path, sep='\t')
         self.list_updated = os.path.getmtime(self.file_path)
 
     def initializeUserAccess(self, logger=None):
-        try:
-            with open(self.file_path) as access_list:
-                for line in access_list:
-                    parsed_line = line.split(":")
-                    if len(parsed_line) != 3:
-                        # skip incorrectly formatted lines
-                        continue
-                    username = parsed_line[0]
-                    project = parsed_line[1]
-                    level = parsed_line[2].strip('\n')
+        project_list = list(self.access_list)[1:]
+        for idx, row in self.access_list.iterrows():
+            username = row['users']
+            if username not in self.user_access_map:
+                self.user_access_map[username] = {}
+                for project in project_list:
                     try:
-                        int(level)
+                        level = int(row[project])
```

Comment:
what is the type of `row[project]` when there's no access level defined? is it empty or an empty string

Reply:
numpy NaN float type and gives ValueError when passed to int()